File: Modul 4/PRAK405/PRAK305-2310817310011-MuhammadRizky.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    scanf("%d %d", &n, &kelipatan);

    int totalSemua = 0;

    for (int i = 1; i <= n; i++) {
        int totalBaris = 0;
        for (int j = i; j >= 1; j--) {
            int jumlah = j * kelipatan;
            totalBaris += jumlah;
            printf("(%d * %d)", j, kelipatan, jumlah);
            if (j > 1) {
                printf(" + ");
            }
        }
        totalSemua += totalBaris;
        printf(" = %d\n", totalBaris);
    }

    printf("%d\n", totalSemua);

    return 0;
}

File: Modul 4/PRAK405/PRAK305-2310817310011-MuhammadRizky.py
n, kelipatan = map(int, input().split())

totalSemua = 0

for i in range(1, n +1):
    totalBaris = 0
    for j in range(i, 0, -1):
        jumlah = j * kelipatan
        totalBaris += jumlah
        print(f"({j} * {kelipatan})", end = "")

        if j > 1:
            print(" + ", end = "")

    totalSemua += totalBaris
    print(f" = {totalBaris}")

print(totalSemua)